### DISCUSSION TITLE
Suspension Of Organization Account And CTO Account Without Any Email Or Similar

### DISCUSSION THREAD

**Discussion by hrittm** [community/community#147726] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

#### Github account (https://github.com/STEAMer-Academy) of STEAMer Academy was suspended suddenly and without any notice, which has seriously affected my work along with Chief Technology Officer's GitHub account (https://github.com/MuntasirSZN)
I submitted a support ticket (https://support.github.com/ticket/personal/0/3150905) through Github's support page but it's been more than days and I still haven't received any response or support.
Entire works have been suspended and our repositories are no longer accessible. There is no way to restore or at least what caused this terrible situation from GitHub, which makes us worried about our creations if this situation continues.
I have read and found that not only me but many people out there are also facing the same situation as me.

**Comment by shinybrightstar** [comment]
Hi @hrimondal  

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
